components/blocks/Features.tsx

```diff
@@ -87,7 +87,7 @@ export function FeatureBlock({ data, index }) {
             {data.media && data.media[0].code && (
               <div className="flex flex-col justify-start items-start">
                 {data.media[0].file && (
-                  <div className="inline-block rounded-t-lg overflow-hidden text-white border-2 border-b-0 border-blue-800 bg-gradient-to-tl from-blue-800 to-blue-900 px-7 py-3 font-tuner">
+                  <div className="inline-block rounded-t-lg overflow-hidden text-white border-2 border-b-0 border-gray-700 bg-gradient-to-tl from-[#333333] to-[#1a1a1a] px-7 py-3 font-tuner">
```

Comment:
do we not have tailwind background colours etc defined yet ;(

Reply:
no we dont use this color scheme anywhere else :(